airgun/views/oscappolicy.py

```diff
@@ -64,8 +65,18 @@ def is_displayed(self):
         )
 
     @View.nested
-    class create_policy(BaseLoggedInView):
-        TAB_NAME = 'Create policy'
+    class deployment_options(BaseLoggedInView):
+        TAB_NAME = 'Deployment Options'
+        next_step = Text("//input[contains(@value, 'Next')]")
+        deploy_by_ = RadioGroup(
```

Comment:
deploy_by_ the extra underscore was it intentional

Reply:
nope :)

Reply:
@jameerpathan111 can you keep this as `deploy_by` only ?

Reply:
yes